minigalaxy/launcher.py

```diff
@@ -167,8 +172,8 @@ def get_scummvm_exe_cmd(game, files):
     return ["scummvm", "-c", scummvm_config]
 
 
-def get_start_script_exe_cmd():
-    return ["./start.sh"]
+def get_start_script_exe_cmd(game):
+    return [os.path.join(game.install_dir, "start.sh")]
```

Comment:
When launching through Minigalaxy. 
But what happens when a .desktop file is created for the game during installation? It's exec command would then be './start.sh'
Under the assumption that the Path entry is set and honored by the respective launcher, it will also work the ./start.sh. I wasn't too sure about the support and behavior there.

Reply:
We do set the Path entry in desktop shortcuts actually.

Reply:
Yeah. i wasn't doubting that. Did see it.

What i meant is that Path is *optional* and depending on the application that actually evaluates these shortcuts, Path may not be parsed and used.
I had stuff like es-de in mind here which do a manual parsing of desktop files. And since there's more than a dozen different desktop environments there's bound to be one that ignores Path, either deliberately or because of a bug.

Reply:
Okay, I guess there is no problem with that.